docs/test-framework-dev-guide.md

```diff
@@ -1,5 +1,19 @@
 # Developer Guide for the Integration and E2E Testing Framework
 
+## Prerequisites
+
+### Dependencies
+
+Go version >= Go 1.19
```

Comment:
Is this necessary? The concern is that by specifying a concrete Go version here it becomes one more place we have to keep it updated whenever we upgrade the Go version recommended to use with the `elastic-agent` repo.

Could we omit this or say that it should be the same Go version as in the [`.go-version`(https://github.com/elastic/elastic-agent/blob/main/.go-version) file at the root of this repository].

Reply:
Good catch!